Dockerfile.ubuntu

```diff
@@ -0,0 +1,19 @@
+# Build Pyradiomics inside Ubuntu 16.04
+
+FROM ubuntu:16.04
+
+MAINTAINER "Daniel Blezek" daniel.blezek@gmail.com
+
+# Install Scipy https://www.scipy.org/install.html
+USER root
+RUN apt-get update && apt-get install -y python-pip python-numpy python-scipy python-nose
```

Comment:
Some of these requirement are redundant. Using `pip install -r requirements.txt` should be sufficient to pull most of the requirements.